package.json

```diff
@@ -47,7 +47,8 @@
     "semver": "^7.5.2",
     "word-wrap": "^1.2.4",
     "@babel/traverse": "7.23.2",
-    "parse-url": "8.1.0"
+    "parse-url": "8.1.0",
+    "domino": "Haringat/domino#2721294c75334ece635dfc72cad0a3dede7391fb"
```

Comment:
Defo want a better way of fixing this but gap-web-ui's `turndown` dependency uses `domino` which uses some deprecated JS syntax that we dont allow due to using `strict` mode. This commit replaces the deprecated JS syntax